### ISSUE TITLE
The interactive elements “arrow” icons cannot be accessed by keyboard in Stack Views tab

### ISSUE THREAD

**Issue by mathisscott** [vmware/clarity#1946]
URL: http://clarity-ks.surge.sh/stackviews

Reproduced Steps:
1. Login Clarity Kitchen Sink with web browser.
2. Go to Stack Views tab.
3. Use the tab key to access all the interactive elements in Stack Views tab.

Actual Results:
The interactive elements “arrow” icons cannot be accessed by keyboard. 
For more details, please refer to “StackViews-CannotBeAccessed.png”.

Expected Results:
All interactive elements can be accessed or activated by keyboard.

Violation of Section 508 Standards: 
1194.21(a)When software is designed to run on a system that has a keyboard, product functions shall be executable from a keyboard where the function itself or the result of performing a function can be discerned textually.

User Impact: Blind.

Screen shot:
1.  StackViews-CannotBeAccessed.png

![image.png](https://images.zenhubusercontent.com/582f4cffb9dc3c6220222f57/0bb817fe-2a87-4847-b393-09dec7972b75)
